### DISCUSSION TITLE
meta.json - root: true not hiding other root

### DISCUSSION THREAD

**Discussion by spiritoftime** [fuma-nama/fumadocs#728] [Q&A] [answered]
Hello! so i have two roots defined in meta.json.
/content/lectures/meta.json
`{ "title": "Lectures", "root": true }`

/content/questions/meta.json
`{"title":"Questions","root": true,"pages":["fourOperations","fractions","geometry","circle","speed","ratio","percentage","algebra","dataAnalysis"]}`

Lecture correctly hides questions
![image](https://github.com/user-attachments/assets/7e738a75-f03d-4b06-ade2-ac61fa7a25fb)
However, questions still displays the lecture 
![image](https://github.com/user-attachments/assets/be1132c2-6b7e-4224-bde0-9a6a5cadb57f)
Was wondering what i did wrong? thank you!
Repo url: https://github.com/spiritoftime/psle-math/tree/main/open-math-prep/content

**Comment by fuma-nama** [comment, marked as answer]
index page won't be considered as the children of folder if they're not listed under `pages`.

That's because some people may prefer to show the rest when not actually clicking into the pages, and it's more user friendly to have an active item in the sidebar to indicate which page user is reading.

I've should have clarified it in the docs, will improve it later.
